product-service/handlers/getAllProducts.js

```diff
@@ -1,8 +1,12 @@
-import { getMockedProducts } from '../utils/createMockData';
+import { query } from './pgClient';
+
+const handler = async (event) => {
+
+    console.log('Incoming event: ', event);
 
-const handler = async () => {
     try {
-      const products = await getMockedProducts();
+      const { rows: products } = await query(`SELECT p.id, p.title, p.description, p.price, p.image, s.count FROM product p INNER JOIN stock s ON p.id = s.product_id`);
```

Comment:
I guess we need `p.*` instead `p.id, p.title, p.description, p.price, p.image`

Reply:
Maybe. I'll leave it as it is for now.